.env.example

```
@@ -2,4 +2,10 @@ AUTH_SECRET=
 AUTH_DISCORD_ID=
 AUTH_DISCORD_SECRET=
 
-ADMINS=
\ No newline at end of file
+ADMINS=
+
+# Pterodactyl Application API key used to auto-tag Birdflop-hosted listings
+# on the server list. Optional; detection is skipped when unset.
+PANEL_API_KEY=
+# Defaults to https://panel.birdflop.com when unset.
+PANEL_URL=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)